auth_service/user_management/user_registry/models/dto.py

```diff
@@ -30,10 +30,9 @@
 class UserStatus(str, Enum):
     """User status enum"""
 
-    REGISTERED = "registered"
-    ACTIVATED = "activated"
-    INACTIVATED = "inactivated"
-    DELETED = "deleted"
```

Comment:
Do we have no need for the DELETED status anymore?

Reply:
We currently do not use REGISTERED AND DELETED, so I have removed them.